packages/xod-arduino/src/Transpiler.rei

```diff
@@ -20,4 +20,7 @@ type liveness =
   | Debug
   | Simulation;
 
-let transpile: (Project.t, PatchPath.t, liveness) => XResult.t(program);
+type xodVariables = Map.String.t(string);
```

Comment:
Sorry for pointing it so late, just recognized the mental conflict: they are not variables, they are constants (from the program point of view), so: `xodVariables` → `xodGlobals`?